powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/v3/TokenService.java

```diff
@@ -125,8 +130,8 @@ public TokenRemoveResponse removeToken(TokenRemoveRequest request, PowerAuthApiA
             response.setTokenId(tokenId);
             return response;
         } catch (Exception ex) {
-            logger.warn("Removing PowerAuth token failed", ex);
-            throw new PowerAuthAuthenticationException(ex.getMessage());
+            logger.warn("Removing PowerAuth token failed, error: {}", ex.getMessage());
+            throw new PowerAuthAuthenticationException("POWER_AUTH_TOKEN_ERROR");
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.